blueprints/app/files/gitignore

```diff
@@ -1,20 +1,14 @@
-# See https://help.github.com/ignore-files/ for more about ignoring files.
-
 # compiled output
 /dist/
-/tmp/
 
 # dependencies
 /node_modules/
 
 # misc
 /.env*
 /.pnp*
-/.sass-cache
 /.eslintcache
-/connect.lock
 /coverage/
-/libpeerconnection.log
```

Comment:
Never seen this being created. I assume it's something that was needed a long time ago.